### PR TITLE
Make gcping responsive

### PR DESCRIPTION
**Pull request by saranshdhingra**
- Adds a degree of responsiveness
- Removes some unused classes and the MDL stylesheet

### Screenshots:
- Tablet Horizontal
image

- Phone vertical
image

- Phone horizontal
image

### CODE REVIEW

**Review by enocom — CHANGES_REQUESTED** [review]
When I squeeze this down, there are a couple of issues I see:

1. the table still has a horizontal scroll
2. the restart button gets squeezed (maybe we could just use the icon for small sizes)
image

**Comment by saranshdhingra** [comment]
> When I squeeze this down, there are a couple of issues I see:
> 
> 1. the table still has a horizontal scroll
> 2. the restart button gets squeezed (maybe we could just use the icon for small sizes)
>    image

I may need to add more breakpoints.
I'll take care of it. Good catch.

**Comment by saranshdhingra** [comment]
@enocom the breakpoints should be adjusted now.
Also check if the following looks OK(_Added some default opacity for the sorting icons on touch screens_).

horizontal

vertical

File: web/src/css/media_queries.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    #how-it-works-dialog-content {
+        a {
+            text-decoration: underline;
+        }
+    }
+
+    .mdc-data-table__sort-icon-button {
+        opacity: 0.2;
+    }
+}
```
Comment (enocom) [inline web/src/css/media_queries.scss:52]: nit: newline

File: web/src/css/media_queries.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  
+      h5 {
+        line-height: 1.5rem;
+      }
+    }
+  
+    .mdc-data-table {
+      .mdc-data-table__header-cell,
+      .mdc-data-table__cell {
+        max-width: 100px;
```
Comment (enocom) [inline web/src/css/media_queries.scss:24]: If I remove this rule, the "Global HTTP load balancer" doesn't wrap and looks better.

File: web/src/css/media_queries.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        white-space: break-spaces;
+        line-height: 1.4;
+  
+        &.result {
+          text-align: right;
+        }
+      }
+      .embedded_region {
+        font-style: italic;
+        margin-top: 10px;
```
Comment (enocom) [inline web/src/css/media_queries.scss:36]: As a rule, let's avoid px rules unless we really need them. This looks better to me as `0.4rem`.

File: web/src/css/media_queries.scss (hunk 2 above)
Comment (enocom) [reply web/src/css/media_queries.scss:24]: Before:

image


After:

image

File: web/src/css/media_queries.scss (hunk 3 above)
Comment (enocom) [reply web/src/css/media_queries.scss:36]: Before:

image

After:

image

**Review by enocom — CHANGES_REQUESTED** [review]
A few minor minor details. This is looking good.

**Comment by enocom** [comment]
@saranshdhingra Once this work is done, what do you think about adding a screenshot to the README so balance out the CLI example?

File: web/src/css/media_queries.scss (hunk 1 above)
Comment (saranshdhingra) [reply web/src/css/media_queries.scss]: > nit: newline

Fixed, but I think I'll soon add this in the eslint rules.

**Comment by saranshdhingra** [comment]
> @saranshdhingra Once this work is done, what do you think about adding a screenshot to the README so balance out the CLI example?

Sure, once this goes merges, I'll add some screens :)

File: web/src/css/media_queries.scss (hunk 1 above)
Comment (enocom) [reply web/src/css/media_queries.scss]: Good idea. Also, there's an editor setting for this. I don't understand why it's not the default, but Linux tools expect it.

**Review by enocom — APPROVED** [review]
Looks great!

**Pull request merged**